pythainlp/spell/__init__.py

```diff
@@ -3,88 +3,14 @@
 Spell checking
 """
 
-from typing import List
+from .pn import NorvigSpellChecker
+from .spell import correct, spell
 
-from .pn import DEFAULT_SPELL_CHECKER, NorvigSpellChecker
+DEFAULT_SPELL_CHECKER = NorvigSpellChecker()
```

Comment:
I'm curious how the instantiation works here. Do we always have this spell checker object when importing `pythainlp`? In other word, when the spell check is instantiated. Can we do lazy instantiation here, i.e. only creating the object when needed?

Reply:
Currently the instance is created immediately at the module loading time. But, yes, lazy instantiation would be better.